<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgInertiaTensor</class>
 <widget class="QDialog" name="CQDlgInertiaTensor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>348</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Inertia Matrix &amp; Center of Mass</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="title">
    <string>Relative to the absolute frame</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Position of the center of mass</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="qqMz">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>85</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqMx">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>25</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqMy">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>55</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>X [m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Y [m]</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Z [m]</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>281</width>
      <height>111</height>
     </rect>
    </property>
    <property name="title">
     <string>Inertia matrix divided by the mass [m^2]</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <widget class="QLineEdit" name="qqI33">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>85</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>85</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI23">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>85</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>25</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>55</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI21">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>25</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI32">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>55</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI22">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>55</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="qqI31">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>25</y>
       <width>76</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QCheckBox" name="qqApplyToSelection">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>260</y>
      <width>281</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Apply to selected shapes</string>
    </property>
   </widget>
  </widget>
  <widget class="QDialogButtonBox" name="qqOkCancel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>310</y>
     <width>301</width>
     <height>32</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqI11</tabstop>
  <tabstop>qqI21</tabstop>
  <tabstop>qqI31</tabstop>
  <tabstop>qqI12</tabstop>
  <tabstop>qqI22</tabstop>
  <tabstop>qqI32</tabstop>
  <tabstop>qqI13</tabstop>
  <tabstop>qqI23</tabstop>
  <tabstop>qqI33</tabstop>
  <tabstop>qqMx</tabstop>
  <tabstop>qqMy</tabstop>
  <tabstop>qqMz</tabstop>
  <tabstop>qqOkCancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
